.packit.yaml

```diff
@@ -84,3 +84,173 @@ jobs:
     post-upstream-clone:
     # builds from master branch should start with 100 release, to have high priority
     - bash -c "sed -i \"s/1%{?dist}/100%{?dist}/g\" packaging/leapp-repository.spec"
+
+- job: tests
+  fmf_url: "https://gitlab.cee.redhat.com/oamg/tmt-plans"
+  fmf_ref: "master"
+  use_internal_tf: True
+  trigger: pull_request
+  targets:
+    epel-7-x86_64:
+      distros: [RHEL-7.9-ZStream]
+  identifier: tests-7.9to8.6
+  tmt_plan: "^(?!.*c2r)(?!.*sap)(?!.*8to9)"
+  tf_post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys"
+  tf_extra_params:
+    environments:
+      - tmt:
+          context:
+            distro: "rhel-7.9"
+  env:
```

Comment:
Also please add `LEAPPDATA_BRANCH=upstream` as env var for every test, like https://github.com/oamg/leapp-repository/blob/master/.github/workflows/tmt-tests.yml#L34

Reply:
done.